global.json

```
@@ -1,6 +1,6 @@
 {
-    "sdk": {
-        "version": "3.1.300",
-        "rollForward": "latestFeature"
-    }
-}
\ No newline at end of file
+  "sdk": {
+    "version": "5.0.101",
```

For a lower bound, I think we can use the first .NET 5 release.

```suggestion
    "version": "5.0.100",
```